stripper/ze_serpentis_temple_p2.cfg

```diff
@@ -1,3 +1,53 @@
+;Make the collision of the snake mouth door a bit larger, so people can't hide in a spot Medusa cant target (might be able to avoid targetting in all doorways, but this is the spot commonly exploited). A better fix would be to allow targetting here, but cba to do that
+add:
+{
+	"targetname" "NoAccess"
+	"classname" "func_wall_toggle"
+	"origin" "-2522 6522 6829"
+	"angles" "0 0 0"
+	"model" "*226"
+	"rendermode" "10"
+}
+
+add:
+{
+	"targetname" "NoAccess"
+	"classname" "func_wall_toggle"
+	"origin" "-2522 6498 6794"
+	"angles" "0 0 0"
+	"model" "*227"
+	"rendermode" "10"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_door"
+		"targetname" "chamber_mouthdoor"
+	}
+	insert:
+	{
+		;Players will never be against the door when it closes due to timing on its inputs, so don't have to worry about them getting stuck in the invisible extended collision
+		"OnOpen" "NoAccess,AddOutput,solid 5,0,-1"
+		"OnFullyClosed" "NoAccess,AddOutput,solid 1,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_door"
+		"targetname" "chamber_mouthdoor"
+	}
+	insert:
+	{
+		"OnOpen" "NoAccess,Disable,,0,-1"
+		"OnFullyClosed" "NoAccess,Enable,,0,-1"
+	}
+}
```

Comment:
Are both of these modify blocks supposed to be here? Just the disable/enable should be sufficient?

Reply:
Disable and enable don't work on func_wall_toggle. That was my first attempt to get it to work that failed and I forgot to remove it after switching to solid modes >.>